samples/msal-node-samples/VanillaJSTestApp/app/scenarios/auth-code-aad/requestConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "authCodeUrlParameters": {
```

Comment:
I would suggest putting these all in the same file if they are part of the same test scenario

Reply:
Done, I condensed the scenario directories into single json files, which lowered the file count considerably. Refactoring the code after that greatly improved readability and cleanliness of this sample, so I think this was a very good suggestion.